test/php/Nominatim/GeocodeTest.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace Nominatim;
+
+require_once(dirname(dirname(__FILE__)).'/../../build/settings/settings.php');
+require_once(dirname(dirname(__FILE__)).'/bootstrap.php');
+
+require_once(CONST_BasePath.'/lib/Geocode.php');
+require_once(CONST_BasePath.'/lib/ParameterParser.php');
+
+class GeocodeTest extends \PHPUnit_Framework_TestCase
+{
+
+
+    protected function setUp()
+    {
+        $this->oDB =& getDB();
+    }
+
+
+    public function testViewbox()
+    {
+        $oGeocode = new Geocode($oDB);
```

Comment:
Do you mean `$this->oDB`? I think you will get away with setting a nullptr (or false) here, if not we really need a mock object, so that the tests don't create a DB connection.